controllers/ibu_controller.go

```diff
@@ -21,6 +21,7 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/go-logr/logr"
```

Comment:
not sure why it was moved here but i think it is mistake

Reply:
this is auto sorted (L34 moved L24)

Reply:
The auto-sort would be done within groups of imports. Looks like this was moved into the group of LCA modules from a group of public modules, then resorted

Reply:
The modules I've written, I've separated into groups of:
* core golang modules
* LCA modules
* external utils/modules
* APIs